vllm/v1/worker/gpu_model_runner.py

```diff
@@ -2770,7 +2773,7 @@ def get_attn_backends_for_layers(
             # they are cached correctly, there will be different objects per
             # layer.
             for layer_name in layer_names:
-                attn_backend = attn_layers[layer_name].get_attn_backend()
+                attn_backend = layers[layer_name].get_attn_backend()
```

Comment:
nit: can you move `attn_layers = get_layers_from_vllm_config(self.vllm_config, Attention)` (line 2760 now) to the line that this variable is first used.